test/data/testSP.py

```diff
@@ -14,8 +14,8 @@
 
 import numpy
 
-from testall import skipForParser
-from testall import skipForLogfile
+from skip import skipForParser
+from skip import skipForLogfile

```

Comment:
> These are giving me an ImportError, unless I change them to relative imports. Same with testSPun and testScan.

Are you sure the import from `skip` is the problem? Could you show the actual error you get here after the latest commits?